proxy/README.md

```diff
@@ -144,94 +110,4 @@ with:
 
 ## Tests
 
-Unit tests can be run with `make test`.
-
-## Example Commands
-
-The following commands can be used to demonstrate the proxy.
-
-This demonstrates proxying a request which has an `application/json` response:
-
-    $ cat examples/posts.json | ./sd-proxy.py ./config-example.yaml
```

Comment:
These docs refer to the Python implementation, and the example payloads were removed alongside it in 0ca17ab1966a80f6af02322697b0e1bcfe2eef58